tee-worker/enclave-runtime/src/rpc/worker_api_direct.rs

```diff
@@ -103,6 +106,32 @@ where
 		Ok(json!(json_value.to_hex()))
 	});
 
+	// author_getEnclaveId
+	let rsa_pubkey_name: &str = "author_getEnclaveId";
+	io.add_sync_method(rsa_pubkey_name, move |_: Params| {
+		let shielding_key_repository = match GLOBAL_SHIELDING_KEY_REPOSITORY_COMPONENT.get() {
+			Ok(s) => s,
+			Err(e) => {
+				let error_msg: String = format!("{:?}", e);
+				debug!("{:?}", e);
+				return Ok(json!(compute_hex_encoded_return_error(error_msg.as_str())))
+			},
+		};
+		let vc_pubkey = shielding_key_repository
```

Comment:
`vc_pubkey` is confusing too, although they are the sam key, I suggest `enclave_signer_account` or `enclave_account`

Reply:
Looks like it's actually the public key tho, so maybe `enclave_signer_public_key`? Also, since that's the value that the method returns, I'd argue that that's what it should be named.